section2/reflection.md

```diff
@@ -1,15 +1,60 @@
 ## Section 2 Reflection
 
 1. What are two points from the Growth Mindset article and/or video that either resonated with you, or were brand new to you?
+    1. "Understanding that experience and practice is essential for growth and success"
```

Comment:
SUCH GOOD REFLECTIONS! Do you feel like you have a way to remind yourself of this perspective, when you are in the moment, super frustrated?